west.yml

```diff
@@ -111,7 +111,7 @@ manifest:
     - name: hostap
       repo-path: sdk-hostap
       path: modules/lib/hostap
-      revision: d82c309c2e7eea5f08bbc1ac9e2f4c4622fbe998
+      revision: v2.11-devel-ncs1-snapshot1
```

Comment:
Do we actually need this tag? `sdk-hostap` doesn't seem to require a rebase (@krish2718 ?).

Reply:
We haven't done a rebase so far, but we have plans to do this, so, we could rebase now if you want, but its optional.

Reply:
it is possible to have no rebase changes.... but for the sake of process this PR brings visibility

Actual rebase PR that I will ask for merge will be: https://github.com/nrfconnect/sdk-nrf/pull/12785

Reply:
Aren't you confusing upmerge (updating with the upstream) with so called "rebase" (history cleanup)? `sdk-hostap` seem to already have linear history (NCS patches on top).

Reply:
It already has a linear history, so, nothing to `rebase`, I was referring to `upmerge` which can be done later.